apps/system/js/card.js

```diff
@@ -350,11 +338,111 @@
   };
 
   Card.prototype._fetchElements = function c__fetchElements() {
+    this.contentNode = this.element.querySelector('.content');
     this.screenshotView = this.element.querySelector('.screenshotView');
     this.titleNode = this.element.querySelector('h1.title');
     this.iconButton = this.element.querySelector('.appIcon');
   };
 
+  Card.prototype._registerEvents = function c__registerEvents() {
+    this.element.addEventListener('touchstart', this);
+    this.element.addEventListener('touchmove', this);
+    this.element.addEventListener('touchend', this);
+    this._eventsRegistered = true;
+  };
+
+  Card.prototype._unregisterEvents = function c__unregisterEvents() {
+    if (!this._eventsRegistered || !this.element) {
+      return;
+    }
+    this.element.removeEventListener('touchstart', this);
+    this.element.removeEventListener('touchmove', this);
+    this.element.removeEventListener('touchend', this);
+    this._eventsRegistered = false;
+  };
+
+  Card.prototype.handleEvent = function(evt) {
+    var verticalY;
+    var tapThreshold = 1;
+    switch (evt.type) {
+      case 'touchstart':
+        evt.stopPropagation();
```

Comment:
Looks like the only way to prevent the cards from scrolling at the same time is to preventDefault() on touchstart :/
But it might be worth asking Kats if there's a way to stop APZC from scrolling at a later point during a touchmove.

(It's already way better than the nested scrollable areas!)